packages/core/src/lib/base_protocol.ts

```diff
@@ -50,6 +50,60 @@ export class BaseProtocol implements IBaseProtocol {
     );
     return peer;
   }
+
+  /**
+   * Retrieves a list of peers based on the specified criteria.
+   *
+   * @param numPeers - The number of peers to retrieve. If you need all available peers, specify a large number.
```

Comment:
```suggestion
   * @param numPeers - The number of peers to retrieve. If you need all available peers, specify a large number.
```

One usually specify `0` for all of something.

Reply:
yes